documentation/api/info.md

```diff
@@ -0,0 +1,50 @@
+# Info - Get
+
+Gets information about the Dotnet-Monitor version. Other configuration information may be included in the future.
+
+## HTTP Route
+
+```http
+GET /info HTTP/1.1
+```
+
+## Host Address
+
+This route is available on all configured addresses.
```

Comment:
Oops, I forgot that everyone would automatically be notified when I pushed - this was just meant to be an intermediary change to save progress since I hadn't started doing any documentation. Sorry about that!